syz-manager/manager.go

```diff
@@ -191,6 +198,25 @@ func RunManager(cfg *mgrconfig.Config, syscalls map[int]bool) {
 	mgr.fresh = len(mgr.corpusDB.Records) == 0
 	Logf(0, "loaded %v programs (%v total, %v deleted)", len(mgr.candidates), len(mgr.corpusDB.Records), deleted)
 
+	Logf(0, "loading legacy repros...")
+	dirs, err := osutil.ListDir(crashdir)
+	for _, dir := range dirs {
```

Comment:
I afraid this again can lead to exponential explosion. We have hub flag in saveRepro and don't send repros from hub back to hub. But after restart we load _all_ repros and send back to hub. This effectively ignores the hub flag.